pylint/epylint.py

```diff
@@ -117,7 +123,32 @@ def lint(filename, options=()):
         return process.returncode
 
 
-def py_run(command_options="", return_std=False, stdout=None, stderr=None):
+@overload
+def py_run(
+    command_options: str = "",
+    return_std: Literal[False] = False,
```

Comment:
Usually overloads should **not** use the explicit default value. Instead use `str = ...` or `Literal[False] = ...`.

Reply:
I'll do a follow-up!